src/core/scala/app/com/linkedin/playparseq/utils/PlayParSeqHelper.scala

```diff
@@ -63,9 +64,18 @@ private[playparseq] abstract class PlayParSeqHelper {
    * @return The Future
    */
   private[playparseq] def bindTaskToFuture[T](task: Task[T], scalaPromise: Promise[T]): Task[T] = {
```

Comment:
Nit: Better named as 'bindTaskToPromise'?  I'll leave it up to you to decide if you want to change it.